tutorials/vue/ssr.md

```diff
@@ -1,12 +1,105 @@
-[Vue has solid support for server-side rendering in Node.js](https://ssr.vuejs.org/guide/#installation). This means that, given a Vue app,
-you can render the app in an Express endpoint using the [`vue-server-renderer` library](https://www.npmjs.com/package/vue-server-renderer).
+[Vue has solid support for server-side rendering in Node.js](https://vuejs.org/guide/scaling-up/ssr.html). This means that, given a Vue app,
+you can render the app in an Express endpoint using the `renderToString` function in the `server-renderer` directory.
 
 Below is an example of rendering a Vue app using [Express](https://www.npmjs.com/package/express).
 
 ```javascript
-[require:Vue.*ssr]
+import express from 'express'
+import { createSSRApp } from 'vue'
+import { renderToString } from 'vue/server-renderer'
+
+const server = express()
+
+server.get('/', (req, res) => {
+  const app = createSSRApp({
+    data: () => ({ count: 1 }),
+    template: `<button @click="count++">{{ count }}</button>`
+  })
+
+  renderToString(app).then((html) => {
+    res.send(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Vue SSR Example</title>
+      </head>
+      <body>
+        <div id="app">${html}</div>
+      </body>
+    </html>
+    `)
+  })
+})
+
+server.listen(3000, () => {
+  console.log('ready')
+})
 ```
 
 Note that the `renderToString()` function returns a promise. Make sure you [handle errors](https://thecodebarbarian.com/introducing-await-js-express-async-support-for-express-apps).
 
-Notice that the above `div` has a `data-server-rendered` attribute. The `data-server-rendered` attribute marks this div for [client-side hydration](https://ssr.vuejs.org/guide/hydration.html). In other words, you can then use the [`$mount()` function](https://vuejs.org/v2/api/#vm-mount) to make your Vue client pick up your static HTML and start reacting to client-side events.
\ No newline at end of file
+## Making the button interactive
+
+The way this is currently set up, the button will not be interactive. 
```

Comment:
The example in this section is a bit hard to set up. Maybe let's just get rid of this section?